package.json

```diff
@@ -27,6 +27,7 @@
     "@types/invariant": "^2.2.34",
     "@typescript-eslint/eslint-plugin": "^4.29.0",
     "@typescript-eslint/parser": "^4.29.0",
+    "aptos": "^1.3.14",
```

Comment:
is it necessary to add aptos here?

Reply:
To run lint successfully, I added it to the devDependencies.

Reply:
got it , thanks for the help here.👍🏻